app/app/Http/Controllers/LocationReportController.php

```diff
@@ -160,4 +166,56 @@ public function showComments(string $location_id)
         
         return view('pages.location_report.comments', $view_data);
     }
+
+    public function addSuggestion(Request $request)
+    {
+        // user needs to sign in before sending suggestions
+        if (!BaseUser::isSignedIn()) {
+            return redirect()->intended('signin');
+        }
+        $user = BaseUser::getDbUser();
+        // validate data from front end
+        $validation_rules = array(
+            'location-id'           => 'required',
+            'location-name'         => 'required',
+            'phone-number'          => '',
+            'url'                   => '',
+            'address'               => 'required'
```

Comment:
Remove the 'required' for address.  Although the vast majority of locations have addresses, some don't so it is still valid to have an empty address.